pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -273,6 +289,16 @@ func RunStartBuild(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra
 	return exitErr
 }
 
+// isBuildLogTimeout returns true if the error indicates timeout getting build
+// logs (default is 10 seconds).
+func isBuildLogTimeout(err error) bool {
```

Comment:
@kargakis this is specific to `StatusError` so I hesitate to make it a generic checker for all timeouts...